articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
+	- [Can Libraries Save the Day?](#can-libraries-save-the-day)
+
+<!-- /TOC -->
+
+# Instructions for Saving Endangered Data: It's time to get decentralized.
+
+## The Situation: Endangered Data Woven into a Precarious Web
 
 **precarious**
 adj. Dangerously lacking in security or stability: _a precarious posture; precarious footing on the ladder._
 adj. Subject to chance or unknown conditions.
 
-It happened with GeoCities. It's happening with climate data. It also happens in little ways every time someone takes a snapshot of a controversial tweet or facebook thread before it gets deleted. Our digital lives are woven into the World Wide Web and that web sits on a precarious foundation. We've been shoring up that foundation with abortive attempts to grab snapshots of content before it changes or disappears. It's time to migrate the web onto resilient, decentralized foundations, starting with the endangered data.
+We keep grabbing pieces of the web before they fall away.  It happened with GeoCities. It's happening with climate data. It also happens in little ways every time someone takes a snapshot of a controversial tweet or Facebook thread before it gets deleted. Our digital lives are woven into the World Wide Web and that web is woven from unstable fibers. We've resisted the degradation of that web with abortive attempts to grab snapshots of content before it changes or disappears, but this isn't enough. It's time to weave a more resilient web by using decentralized techniques. We can start by using these techniques to catch and hold endangered data.
 
+The World Wide Web is precarious. It's a network of information, discussions, data, and analysis that relies on HTTP links to maintain connections between the pieces. Those links are unreliable because the content they point to is unstable. If you rely on an HTTP service as the authoritative source of data you're building on a precarious foundation. Next time you feel the pain of this precarity, I encourage you to take it as a sign that it's time for you to embrace the decentralized web.
 
-The World Wide Web is precarious. Any network of information, discussions, data, or analysis that relies on HTTP links is built on a precarious foundation. Likewise, if you rely on an HTTP service as the authoritative source of data you're building on a precarious foundation. Next time you feel the pain of this precarity, I encourage you to take it as a sign that it's time for you to embrace the decentralized web.
+When you hear about people scrambling to "back up" data from services that are about to be turned off, or when you find yourself backing up that content yourself, I want you to ask “Why are we still using location-based links to point to stuff on the web? Links shouldn't point to locations where the content _might_ be found. They should point to the content itself, which can exist in many locations at once.”
```

Comment:
Where will this be viewed? It might be worth explaining location-based vs content-based pointers, before asking people to think about them. That would help inform new readers who don't know that you _can_ point to data itself, using a hash. This is something that I suspect may be unfamiliar to a lot of people; they're used to IDs, but they're not used to those IDs being cryptographic hashes. They may also wonder if those hashes have to refer to the data that they represent - why would they? Most hashes don't. Instead, I think, it might be beneficial to explain that a content-based FS would enable us to know, with certainty, that hashes point to data, that the hash is representative of the data, and that, given the hash, you can find the data. Otherwise, there are a few steps that the reader has to be aware of for your call to arms to work well.